lib/pbench/test/unit/server/conftest.py

```diff
@@ -289,6 +295,11 @@ def provide_metadata(attach_dataset):
     test = Dataset.attach(controller="node", name="test")
     Metadata.setvalue(dataset=drb, key="user.contact", value="me@example.com")
     Metadata.setvalue(dataset=drb, key=Metadata.DELETION, value="2022-12-25")
+    Metadata.setvalue(
+        dataset=drb,
+        key="server.index-map",
+        value={"unit-test.v5.result-data-sample.2020-08": ["random_md5_string1"]},
```

Comment:
Again, for some semblance of reality, the "random_md5_string1" shouldn't be the `run_id` MD5. That value goes under the `unit-test.v6.run-data.2020-08` index.

Reply:
To reiterate: it's the `<prefix>.<version>.result-data-sample.<date>` pattern of the index you need to match, not the values in its list of documents.